ledger/acctupdates.go

```diff
@@ -164,7 +164,10 @@ type accountUpdates struct {
 	cachedDBRound basics.Round
 
 	// deltas stores updates for every round after dbRound.
-	deltas []ledgercore.AccountDeltas
+	deltas []ledgercore.StateDelta
+
+	// deltas stores updates for every round after dbRound.
+	// deltas []ledgercore.AccountDeltas
```

Comment:
I didn't realize you had made this change. Very nice!

nit: remove dead code
```suggestion
	deltas []ledgercore.StateDelta
```